db/migrate/20160705110711_add_start_time_to_event_schedule.rb

```diff
@@ -0,0 +1,5 @@
+class AddStartTimeToEventSchedule < ActiveRecord::Migration
```

Comment:
Because I forgot it in the previous one. Should I rollback the previous one and add it in that one?